lib/web/app/match.go

```diff
@@ -133,6 +133,12 @@ func MatchAll(matchers ...Matcher) Matcher {
 	}
 }
 
+// ClusterGetter provides a means to retrieve all connected
+// Teleport clusters - either local or remote.
+type ClusterGetter interface {
```

Comment:
nit: maybe rename ClusterLister to differentiate from the 'real' getter in lib/proxy/router.go?